Package.swift

```diff
@@ -43,6 +43,10 @@ let package = Package(
             name: "KafkaFoundationCompat",
             targets: ["KafkaFoundationCompat"]
         ),
+        .library(
```

Comment:
@FranzBusch are you ok with exposing `KafkaTestUtils` as a `.library`?

Alternatively, we could vend the internal testing functions as part of the `Kafka` package and mark them as `@_spi(Internal)`. Then `KafkaTests` and `Benchmarks` would `@_spi(Internal) import Kafka` to access `createUniqueTestTopic` and `deleteTopic` et. al.